views/scores.pug

```diff
@@ -6,10 +6,15 @@ html
     link(href="https://fonts.googleapis.com/css?family=Lexend+Deca|Press+Start+2P&display=swap" rel="stylesheet")
     script(defer)
       include js/randomColorsForScores.js
+    script(defer)
+      include js/filterScores.js
   body
     #high_scores_container
       h1 High Scores - #{prettyGameName}
-      .scores_list
+      #score_filter
+        .icon &#128269;
+        input(type="text" placeholder="Filter scores")#score_filter_input
```

Comment:
Just noticed this but I've been putting class and ID names before attributes as done in [Pug documentation](https://pugjs.org/language/attributes.html) - might be work standardizing eventually

Reply:
Oops, I've been doing it this way for everything. Will standardize in a separate PR later.